t/22-svirt.t

```diff
@@ -1034,4 +1034,136 @@ subtest 'get_wait_still_screen_on_here_doc_input' => sub {
+    is((-e $file_xz), 1, "mock image $file_xz created ok.");
+    is((-e $file), 1, "mock image $file exists.");
+
+    my $file_i = path($file)->copy_to($my_test_dir_iso);
+    my $file_h = path($file)->copy_to($my_test_dir_hdd);
+    my $file_xz_i = path($file_xz)->copy_to($my_test_dir_iso);
+    my $file_xz_h = path($file_xz)->copy_to($my_test_dir_hdd);
+
+    # dest: file shall be missing in destination dir. to trigger reload
+    path($vmware_openqa_datastore, $input_file)->remove;
+    # dest: xz file hall be missing in destination dir. to trigger reload
+    path($vmware_openqa_datastore, $input_file_xz)->remove;
+
+    my $svirt = consoles::sshVirtsh->new('svirt');
+    my $console_mock = Test::MockModule->new('consoles::sshVirtsh');
+    my @last_ssh_commands = ();
+    my $out2;
+
+    $console_mock->redefine(run_cmd => sub ($self, $cmd, %args) {
+            push @last_ssh_commands, $cmd;
+            # run shell script in local dir.
+            my $out;
+            $out = qx($cmd);
+            $out;
+    });
+
+    $out2 = $svirt->provide_image_vmware_in_ds($input_file_xz, $vmware_openqa_datastore, backingfile => 1);
+    like $last_ssh_commands[0], qr/\/hdd\/$input_file_xz\s/, 'Checking image management script with xz- 5a';
+    like $out2, qr/$vmware_openqa_datastore\/$input_file/, 'Checking image management output with xz- 5b';
+
+    path($vmware_openqa_datastore, $input_file)->remove;
+
+    $out2 = $svirt->provide_image_vmware_in_ds($input_file, $vmware_openqa_datastore, backingfile => 1);
+    like $last_ssh_commands[1], qr/\/hdd\/$input_file\s/, 'Checking image management script without xz- 6a';
+    like $out2, qr/$vmware_openqa_datastore\/$input_file/, 'Checking image management output without xz- 6b';
+
+    path($vmware_openqa_datastore, $input_file)->remove;
+
+    $out2 = $svirt->provide_image_vmware_in_ds($input_file, $vmware_openqa_datastore, backingfile => 0);
+    like $last_ssh_commands[2], qr/\/iso\/$input_file\s/, 'Checking image management script without xz- 7a';
+    like $out2, qr/$vmware_openqa_datastore\/$input_file/, 'Checking image management output without xz- 7b';
+
+    path($vmware_openqa_datastore, $input_file)->remove;
+
+    $out2 = $svirt->provide_image_vmware_in_ds($input_file_xz, $vmware_openqa_datastore);
+    like $last_ssh_commands[3], qr/\/iso\/$input_file_xz\s/, 'Checking image management script with xz- 8a';
+    like $out2, qr/$vmware_openqa_datastore\/$input_file/, 'Checking image management output with xz- 8b';
+
+    # $out2 = $svirt->provide_image_vmware_in_ds($input_file, $vmware_openqa_datastore);
+    # like $last_ssh_commands[4], qr/\/iso\/$input_file_xz\s/, 'Checking image management script with xz- 9a';
+    # like $out2, qr/$vmware_openqa_datastore\/$input_file/, 'Checking image management output with xz- 9b';
+
+};
+
+
+subtest 'Routine consoles::sshVirtsh::provide_image_vmware_in_ds' => sub {
+    set_var(VMWARE_NFS_DATASTORE_DEBUG => 1);
```

Comment:
this might be breaking the test as it produces some unhandled output? not sure

Reply:
No, this is not impacting. It is not a new parameter,also used in another routine to simply add for debugging "set -x" to the shell code and check its flow in the log, when needed.

Reply:
Ah, Now it is more clear the question 🙂.

Reply:
Still `not ok - unhandled output found`